cli/yarn.lock

```diff
@@ -1033,6 +1033,7 @@
     ethereumjs-tx "^1.3.7"
     invariant "^2.2.2"
     isomorphic-ws "^4.0.1"
+    ledger-cosmos-js "^2.1.7"
```

Comment:
We use the bundled one in cli too technically. That should clean up the rest of the diff in the lock file